pointgrey_camera_driver/cmake/download_spinnaker

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I don't know how much you/we care, but the more proper solution here is probably to drop this shebang altogether and just change this line:

https://github.com/clearpathrobotics/pointgrey_camera_driver/blob/master/pointgrey_camera_driver/cmake/DownloadSpinnaker.cmake#L11

To include `${PYTHON_EXECUTABLE}`, so that the script runs with whatever python catkin is using.

Reply:
The script is now only python3 so running under python2 would fail.  It's just the changes in the urllib imports so we could easily add that support but I assumed since its our fork that we would want to just force everything to python3.

We've diverged this file from upstream to allow for internal downloads.  I'm not sure if you we want to fix the upstream download script to be dual support?

Reply:
If it's Py3 only then this is fine. As you say, we've diverged quite a bit from upstream at this point.